legend-engine-xts-graphQL/legend-engine-xt-graphQL-query/src/main/java/org/finos/legend/engine/query/graphQL/api/GraphQL.java

```diff
@@ -55,23 +55,23 @@ protected PureModel loadSDLCProjectModel(MutableList<CommonProfile> profiles, Ht
         PureModelContextPointer pointer = new PureModelContextPointer();
         pointer.sdlcInfo = sdlcInfo;
 
-        Subject subject = ProfileManagerHelper.extractSubject(profiles);
+        Subject subject = identity.getSubjectFromIdentity();
```

Comment:
We should fix this in the next pass. There's no guarantee we have a subject here